packages/emotion/native/Css.ml

```diff
@@ -107,6 +107,7 @@ let render_declarations (rules : rule list) =
 let is_at_rule selector = String.contains selector '@'
 let is_a_pseudo_selector selector = String.starts_with ~prefix:":" selector
 let starts_with_at selector = String.starts_with ~prefix:"@" selector
+let starts_with_double_dot selector = String.starts_with ~prefix:":" selector
```

Comment:
Dumb question: what is the difference between `starts_with_double_dot` and `is_a_pseudo_selector`

Reply:
Just saw you remove it here: https://github.com/davesnx/styled-ppx/pull/483/files#diff-fae8d1b5ad79dc3019cf34c19b6577b5e210c318ad4fe23e9816de4ad4c0a4daL140

Should we do the same here?

Reply:
I'm removing support for Css in monorepo and will eventually remove it from styled-ppx. That's why Css isn't at pair with CssJs (where some bugfix got fixed only in CssJs)